packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_account_account_to_utxos.test.ts

```diff
@@ -0,0 +1,143 @@
+import BigNumber from 'bignumber.js'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { OP_CODES } from '@defichain/jellyfish-transaction'
+import { AccountToUtxos } from '@defichain/jellyfish-transaction/dist/script/defi/dftx_account'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { utxosToAccount } from '@defichain/testing'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder } from '../../src'
+import {
+  findOuts,
+  fundEllipticPair,
+  sendTransaction
+} from '../test.utils'
+import { Bech32, HASH160 } from '@defichain/jellyfish-crypto'
+
+const container = new MasterNodeRegTestContainer()
+let providers: MockProviders
+let builder: P2WPKHTransactionBuilder
+let jsonRpc: JsonRpcClient
+
+beforeAll(async () => {
+  await container.start()
+  await container.waitForReady()
+  await container.waitForWalletCoinbaseMaturity()
+  jsonRpc = new JsonRpcClient(await container.getCachedRpcUrl())
+  providers = await getProviders(container)
+
+  // Prep 1000 DFI UTXOS for testing
+  await container.waitForWalletBalanceGTE(1001)
+})
+
+afterAll(async () => {
+  await container.stop()
+})
+
+beforeEach(async () => {
+  await providers.randomizeEllipticPair()
+  await container.waitForWalletBalanceGTE(11.1)
+  builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic)
+
+  // Fund 10 DFI TOKEN
+  await providers.setupMocks() // required to move utxos
```

Comment:
`await providers.setupMocks()` should always be placed below so that it does not accidentally move funds before the test starts, but the way you wrote the test it would do that accidentally.

Created a ticket https://github.com/DeFiCh/jellyfish/issues/274 to tackle it in the future.